test/Microsoft.ML.Sweeper.Tests/Microsoft.ML.Sweeper.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
+    <PackageReference Include="xunit" Version="2.3.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
+    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
+  </ItemGroup>
```

Comment:
I'd try to adhere to the same structure of the other test projects. 

There are a few things, like an CORECLR constant, PlatformTarget set to AnyCPU, and i am not sure about the Test.Sdk will blend with the rest. 

Take a look at Microsoft.ML.Predictor.Tests.csproj

Reply:
I've amended the csproj as you suggested